UnityProject/Assets/Scripts/Managers/GameManager.cs

```diff
@@ -229,28 +231,35 @@ private void LoadConfig()
 		Physics2D.simulationMode = SimulationMode2D.Update;
 	}
 
+	private void Start()
+	{
+		if ( CustomNetworkManager.IsServer ) UpdateManager.Add(UpdateMinutes, 1f);
+	}
+
 	private void OnEnable()
 	{
 		SceneManager.activeSceneChanged += OnSceneChange;
 		UpdateManager.Add(CallbackType.UPDATE, UpdateMe);
 		EventManager.AddHandler( Event.Cleanup, ClientCleanupInbetweenScenes );
 		EventManager.AddHandler( Event.CleanupEnd, ClientCleanupEndRoundCleanups );
 		EventManager.AddHandler( Event.PostRoundStarted, ClientRoundStartCleanup );
-
-
 	}
 
 	private void OnDisable()
 	{
 		SceneManager.activeSceneChanged -= OnSceneChange;
 		UpdateManager.Remove(CallbackType.UPDATE, UpdateMe);
+		if ( CustomNetworkManager.IsServer ) UpdateManager.Remove(CallbackType.PERIODIC_UPDATE, UpdateMinutes);
 		EventManager.RemoveHandler( Event.Cleanup, ClientCleanupInbetweenScenes );
 		EventManager.RemoveHandler( Event.CleanupEnd, ClientCleanupEndRoundCleanups );
 		EventManager.RemoveHandler( Event.PostRoundStarted, ClientRoundStartCleanup );
-
 	}
 
-
+	private void UpdateMinutes()
```

Comment:
why is it called ``UpdateMinutes`` if it is updating every second?

Reply:
oh, whoops. It should update every 60 seconds, not every second.

Reply:
it is probably better to handle it in seconds. Doesn't your clock also display seconds?

Reply:
what clock?

Reply:
the wallmounted clocks. They are supposed to give the time but they just give you the value you set at round start. If you use StationTime instead it is going to actually work

Reply:
well you added a new implementation to StationTime on **this pr**

Reply:
Both are different from each other.

`StationTime` is the oldest value used to represent how long a round has been going.
`StationTimeInMinutes` is an integer value that keeps track of the round's time while avoiding a bug with `DateTime` that gives negative numbers once 24 hours pass.

Clocks use neither, as they use their own role-play compliant variables that allow players to see time as if it's in the year 2500+ and are there to serve a completely different purpose and future use-cases.

Honestly, `StationTime` should be re-named to `RoundTime` but this confusion wouldn't be an issue even if the naming was bad if we had comments explaining the difference between all of them, but we'd rather have clean code that is hard to follow than ugly code that is detailed and doesn't waste anyone's time.

Reply:
rename it to RoundTime.